packages/api/tests/endpoints/collections/active-collections.js

```diff
@@ -0,0 +1,161 @@
+const {
+  recursivelyDeleteS3Bucket
+} = require('@cumulus/aws-client/S3');
+const { randomId } = require('@cumulus/common/test-utils');
+const models = require('../../../models');
+const bootstrap = require('../../../lambdas/bootstrap');
+const indexer = rewire('../../../es/indexer');
+const {
+  createFakeJwtAuthToken,
+  fakeCollectionFactory,
+  fakeGranuleFactoryV2,
+  setAuthorizedOAuthUsers
+} = require('../../../lib/testUtils');
+const { Search } = require('../../../es/search');
+const assertions = require('../../../lib/assertions');
+
+process.env.AccessTokensTable = randomId('accessTokensTable');
+process.env.CollectionsTable = randomId('collectionsTable');
+process.env.GranulesTable = randomId('granulesTable');
+process.env.stackName = randomId('stackName');
+process.env.system_bucket = randomId('systemBucket');
+process.env.TOKEN_SECRET = randomId('tokenSecret');
+
+// import the express app after setting the env variables
+const { app } = require('../../../app');
+
+const esIndex = randomId('esindex');
+let esClient;
+
+let jwtAuthToken;
+let accessTokenModel;
+let collectionModel;
+let granuleModel;
+
+test.before(async () => {
+  const esAlias = randomId('esAlias');
+  process.env.ES_INDEX = esAlias;
+  await bootstrap.bootstrapElasticSearch('fakehost', esIndex, esAlias);
+
+  await awsServices.s3().createBucket({ Bucket: process.env.system_bucket }).promise();
+
+  collectionModel = new models.Collection({ tableName: process.env.CollectionsTable });
+  await collectionModel.createTable();
+
+  granuleModel = new models.Granule({ tableName: process.env.GranulesTable });
+  await granuleModel.createTable();
+
+  const username = randomId('username');
+  await setAuthorizedOAuthUsers([username]);
+
+  accessTokenModel = new models.AccessToken();
+  await accessTokenModel.createTable();
+
+  jwtAuthToken = await createFakeJwtAuthToken({ accessTokenModel, username });
+  esClient = await Search.es('fakehost');
+
+  await Promise.all([
+    indexer.indexCollection(esClient, fakeCollectionFactory({
+      name: 'coll1',
+      version: '1'
+    }), esAlias),
+    indexer.indexCollection(esClient, fakeCollectionFactory({
+      name: 'coll2',
+      version: '1'
+    }), esAlias),
+    indexer.indexGranule(esClient, fakeGranuleFactoryV2({ collectionId: 'coll1___1' }), esAlias),
+    indexer.indexGranule(esClient, fakeGranuleFactoryV2({ collectionId: 'coll1___1' }), esAlias)
+  ]);
+
+  // Indexing using Date.now() to generate the timestamp
+  const stub = sinon.stub(Date, 'now').returns((new Date(2020, 0, 29)).getTime());
+
+  await Promise.all([
+    indexer.indexCollection(esClient, fakeCollectionFactory({
+      name: 'coll3',
+      version: '1',
+      updatedAt: new Date(2020, 0, 29)
+    }), esAlias),
+    indexer.indexGranule(esClient, fakeGranuleFactoryV2({
+      updatedAt: new Date(2020, 1, 29),
+      collectionId: 'coll3___1'
+    }), esAlias)
+  ]);
+
+  stub.restore();
```

Comment:
You should surround this `await` with a `try/finally` block and place `stub.restore` in the `finally` block since stubbing `Date.now` is probably used all over the place and you want to make the stub gets reverted before any operations in the test happen. Otherwise you might get tests failing/passing when they shouldn't